app.rb

```diff
@@ -12,13 +12,29 @@
 require './lib/apruve_overrides'
 require './helpers/demo_order_helper'
 require './helpers/oauth_helper'
+require 'sinatra/flash'
+require 'aws-sdk'
+enable :sessions
+use Rack::Logger
+
+require 'aws/s3'
+AWS::S3::Base.establish_connection!(
+    :access_key_id => 'AKIASFWBQ35QI6KVTWQC',
```

Comment:
FYI, it's a security issue to post the AWS keys in this repo, since it is public. This account has been disabled and replaced.

Typically we add a environment settings for private keys.

Reply:
Thank you!